.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+**Steps to reproduce**
```

Comment:
The text is bold, I don't think any of these are headers. It will look strange if these are headers IMO.